iOS/Flows/Conversation/Conversation Header/MembersCollectionViewLayout.swift

```diff
@@ -51,42 +51,41 @@ class MembersCollectionViewLayout: UICollectionViewCompositionalLayout {
     }
     
     override open func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
-        #warning("fix layout")
-        return super.layoutAttributesForElements(in: rect)
-//        guard let attributes = super.layoutAttributesForElements(in: rect) as? [MemberCellLayoutAttributes] else {
-//            return nil
-//        }
-//        return attributes.map({ attribute in
-//            let copy = attribute.copy() as! MemberCellLayoutAttributes
-//            return self.transformLayoutAttributes(copy)
-//        })
+        guard let attributes = super.layoutAttributesForElements(in: rect) as? [MemberCellLayoutAttributes] else {
```

Comment:
Almost fixed this! it basically works but I couldn't get it to animate in and out correctly.